release/Vagrantfile

```diff
@@ -76,7 +78,7 @@ Vagrant.configure(2) do |config|
     apt-get install -y default-jdk git subversion xmlstarlet zip unzip language-pack-en \
                        vim-nox gnupg2 gnupg-agent pinentry-curses golang rpm mmv haveged
     mkdir -p /opt
-    MAVEN_VERSION=3.3.39
+    MAVEN_VERSION=3.3.3
```

Comment:
why not `apt-get install maven`

Reply:
Because the latest default install of maven on Ubuntu doesn't work with our current build.